Assets/Scripts/Game Controllers/CardManager.cs

```diff
@@ -0,0 +1,47 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CardManager : MonoBehaviour
+{
+    [SerializeField]
+    private PlayerController player;
+
+    public bool PlayCard(string card)
```

Comment:
This could be renamed to `TryPlayCard` because it returns a boolean indicating success, but it doesn't matter much.